Added remixer field

Bootjewolf opened the pull request:
## Description

Fixes #4428

Adds the remixer field from the musicbrainz database to the beets library. I haven't added any documentation since I didn't know if it is necessary for the changes I made.

## To Do

- [x] Documentation. (If you've add a new command-line flag, for example, find the appropriate page under `docs/` to describe it.)
- [x] Changelog. (Add an entry to `docs/changelog.rst` near the top of the document.)
- [x] Tests. (Encouraged but not strictly required.)

sampsyo commented on beets/autotag/mb.py:
```diff
@@ -202,6 +202,19 @@ def _flatten_artist_credit(credit):
 
 
+def _get_remixer_names(relations):
+    """ Given a list representing the artist relationships extract the names of
```
```suggestion
    """Given a list representing the artist relationships extract the names of
```

sampsyo commented on beets/autotag/mb.py:
```diff
@@ -202,6 +202,19 @@ def _flatten_artist_credit(credit):
     )
 
 
+def _get_remixer_names(relations):
```
Maybe we should very slightly generalize this for future-proofing… we could make it take the relation type as a parameter, and make it default to `'remixer'` for now? I can imagine wanting to collect other relationships in a similar way.

sampsyo reviewed (commented):
Wahoo; looks awesome!! I just have two very tiny comments.

Bootjewolf commented:
I updated the PR to include you feedback. I also noticed I introduced a failing test by changing the order of the parameters of the _make_track function so I added the remixer parameter as the last parameter.

sampsyo commented:
Awesome!! Seems like everything's in order. Thanks again!!

Bootjewolf commented:
I really liked working on beets since I also use it myself. This will probably not be my last PR for beets.
